core/src/banking_stage.rs

```diff
@@ -2000,19 +2000,9 @@ impl BankingStage {
         packet_count_upperbound: usize,
     ) -> Result<(Vec<PacketBatch>, Option<SigverifyTracerPacketStats>), RecvTimeoutError> {
         let start = Instant::now();
-        let mut aggregated_tracer_packet_stats_option: Option<SigverifyTracerPacketStats> = None;
-        let (mut packet_batches, new_tracer_packet_stats_option) =
```

Comment:
we set aggregated_tracer_packet_stats_option to None here, and then get a new option back from recv_timeout.

Reply:
hmm this seems like a bad merge, should be aggregating the metrics in this loop: https://github.com/solana-labs/solana/blob/a2e7bdf50ac5e1d4c633f64f6362028b4164c003/core/src/banking_stage.rs#L2017-L2032